share/docker/run-tests.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function headline(){
+  echo -e "\e[1;34m#################################################################"
+  echo -e "##### $1 \e[1;37m"
+  echo -e "\e[1;34m#################################################################\e[0m"
+}
+
+set -e
+
+headline "Run Java tests"
+cd /avro/lang/java
+./build.sh test
+
+headline "Run Python2 tests"
+cd /avro/lang/py
+./build.sh test
+
+headline "Run Python3 tests"
+cd /avro/lang/py3
+./build.sh test
+
+headline "Run C tests"
+cd /avro/lang/c
+./build.sh test
+
+headline "Run C++ tests"
+cd /avro/lang/c++
+# The current cpp tests are failing:
+# https://issues.apache.org/jira/projects/AVRO/issues/AVRO-2230
```

Comment:
It's possible that upgrading to C++11 will also fix this issue. That may be preferable to my fix in #335 
Up to you if you want to give it a shot in this PR. https://stackoverflow.com/a/25639772/418413

Reply:
My C++ skills are not so great. After university I haven't done much, so I'm not familiar with the different versions. My preference would be to update #335 after this one gets merged and then enable the test again 👍